arm-network/2016-06-01/swagger/network.json

```diff
@@ -4251,6 +4251,49 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworks/{virtualNetworkName}/CheckIpAddressAvailability": {
+      "get": {
+        "operationId": "CheckIpAddressAvailability",
```

Comment:
This should be "VirtualNetworks_CheckIpAddressAvailability" as the operation is happening on a virtualNetwork.
CheckDnsNameAvailability is at a higher level if you compare the path for both the operations:
"/subscriptions/{subscriptionId}/providers/Microsoft.Network/locations/{location}/CheckDnsNameAvailability"